CMakeLists_files.cmake

```diff
@@ -160,6 +160,7 @@ list (APPEND TEST_DATA_FILES
 list (APPEND EXAMPLE_SOURCE_FILES
   examples/finitevolume/finitevolume.cc
   examples/mirror_grid.cpp
+  examples/griditer.cpp
```

Comment:
unconditional openmp usage. either make it conditional in build system or adjust the test.

Reply:
Adjusted the test so I did not have to mess with the build system.